src/Discord.Net.Rest/DiscordRestApiClient.cs

```diff
@@ -235,6 +235,7 @@ private async Task LogoutInternalAsync()
             options.BucketId = bucketId;
 
             string json = payload != null ? SerializeJson(payload) : null;
+            Console.WriteLine($"Sending JSON....\n{json}");
```

Comment:
I feel like this is a forgotten line for debugging purposes? But it could be useful if it was wrapped in preprocessor #if check with something like DEBUG_REST

Reply:
thats exactly what it was. removed.